### PR TITLE
added test case for None columns

### PR DESCRIPTION
**Pull request by drahc1R**
added edge case test for a `None` type column in uncorrelated dataset generation

### CODE REVIEW

**Review by taylorfturner — APPROVED** [review]

**Review by taylorfturner — APPROVED** [review]

**Review by ksneab7 — APPROVED** [review]

File: synthetic_data/generators.py [hunk 1]
```diff
@@ -108,7 +108,7 @@ def _generate_uncorrelated_column_data(self, num_samples):
 
             if not generator_name:
                 logging.warning(
-                    f"Generator is {generator_name}. Null entries are not implemented."
+                    f"Generator of type {generator_name} is not implemented."
```
Comment (taylorfturner) [inline synthetic_data/generators.py:111]: Issue #326
